sdcm/utils/data_validator.py

```diff
@@ -207,7 +207,12 @@ def view_names_for_updated_data(self):
             mvs_names = self.get_view_name_from_profile(self.SUFFIX_FOR_VIEW_AFTER_UPDATE, all_entries=True)
             self._mvs_for_updated_data = [view_name.replace(self.SUFFIX_FOR_VIEW_AFTER_UPDATE, '')
```

Comment:
Please fix the problem right here, in the following manner
```suggestion
            self._mvs_for_updated_data = set(view_name.replace(self.SUFFIX_FOR_VIEW_AFTER_UPDATE, '') for view_name in mvs_names)
```

Reply:
fixed